data_steward/tools/import_rdr_omop.sh

```diff
@@ -58,7 +58,7 @@ export GOOGLE_CLOUD_PROJECT="${app_id}"
 export BIGQUERY_DATASET_ID="${OUTPUT_DATASET}"
 
 today=$(date '+%Y%m%d')
-RDR_DATASET="${today}_rdr"
+RDR_DATASET="rdr${today}"
```

Comment:
Can the `RDR_DATASET` be changed to a user parameter? I think we want the date to be based on when the RDR export happened rather than the current date, and it's best to let the user decide the name. The parameter could be named `rdr_dataset` or `rdr_dataset_id`.